common/src/main/scala/uk/ac/wellcome/storage/HybridRecordEnricher.scala

```diff
@@ -0,0 +1,38 @@
+package uk.ac.wellcome.storage
+
+import shapeless.ops.hlist.{Collect, Prepend, Zip}
+import shapeless.{Id => ShapelessId, _}
+import uk.ac.wellcome.models.Id
+
+trait HybridRecordEnricher[T] {
```

Comment:
What does it mean to enrich a record?

Reply:
In this case it's appending the fields from the metadata type handed in.